transformers.md

```diff
@@ -233,15 +235,15 @@ If we neglect the cost of dot-product attention for shorter-context training, th
 
 $$
 \begin{align*}
-18BTDF + 24BTDNH = 6 *BT * (3DF + 4DNH) \\ = 6 * \textrm{num tokens} * \textrm{parameter count}
+(18BTDF + 12BTD(N+K)H)L = 6 *BT * (3DF + 2D(N+K)H)L \\ = 6 * \textrm{num tokens} * \textrm{parameter count}
 \end{align*}
 $$
 
-Leading to a famous rule of thumb for estimating Transformer FLOP count, ignoring the attention FLOPs. (Unembedding is another simple matmul with $6BSEV$ FLOPs and $EV$ params, and follows the same rule of thumb.)
```

Comment:
I don't think we define EV. I assume we use E=D here, so maybe we can just do that?